doc/how-to-run-in-windows.rst

```diff
@@ -0,0 +1,143 @@
+How to use Cabal in Windows
+===========================
+
+This document describes how to use Cabal in a Windows system. See the
+:ref:`Further reading` section for some other references that might provide some
+more explanations. For a TL;DR, jump to the :ref:`Complete configuration`.
+
+Install the Haskell environment
+-------------------------------
+
+The recommended way of setting up a Haskell environment in Windows is by using
+`GHCup <https://www.haskell.org/ghcup/>`_. Follow the steps outlined in its
+webpage to install at least GHC and Cabal. GHCup will install its own MSYS2
+system in your computer unless told not to do so, refer to `its documentation

```

Comment:
I’d replace the comma here (before ‘refer’) with a colon. (See [run-on sentence](https://en.wikipedia.org/wiki/Sentence_clause_structure#Run-on_sentences) for details.)

Reply:
Done